cli/src/commands/util.rs

```diff
@@ -34,15 +34,21 @@ pub(crate) enum UtilCommand {
     ConfigSchema(UtilConfigSchemaArgs),
 }
 
+// TODO: rustfmt is disabled for the entire struct. Can we make it affect the
+// docstring only?
```

Comment:
it's kinda hacky, but this works:
```diff
diff --git a/cli/src/commands/util.rs b/cli/src/commands/util.rs
index 69f3f914..172c8551 100644
--- a/cli/src/commands/util.rs
+++ b/cli/src/commands/util.rs
@@ -44,11 +44,11 @@ pub(crate) enum UtilCommand {
 /// - **bash**: `source <(jj util completion)`
 /// - **fish**: `jj util completion --fish | source`
 /// - **zsh**:
-///     ```shell
-///     autoload -U compinit
-///     compinit
-///     source <(jj util completion --zsh)
-///     ```
+#[doc = "     ```shell
+     autoload -U compinit
+     compinit
+     source <(jj util completion --zsh)
+     ```"]
 #[derive(clap::Args, Clone, Debug)]
 #[command(verbatim_doc_comment)]
 pub(crate) struct UtilCompletionArgs {

```

Reply:
I suppose we could specify the entire docstring as an explicit `doc` attribute. I'm going back and forth on whether I like the idea.

Reply:
I guess it looks OK. :) Thanks!